core/base/context.go

```diff
@@ -51,7 +79,10 @@ type SentinelOutput struct {
 }
 
 func newEmptyOutput() *SentinelOutput {
-	return &SentinelOutput{}
+	return &SentinelOutput{
+		LastResult:  nil,
+		Attachments: make(map[interface{}]interface{}),
+	}
 }
 
 // Reset init EntryContext,
```

Comment:
The reset method should be updated for new fields.

Reply:
Right